pkg/frontend/openshiftcluster_get_effectiveroutetable.go

```diff
@@ -0,0 +1,69 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v6"
+)
+
+func (f *frontend) _getOpenshiftClusterEffectiveRouteTable(ctx context.Context, w http.ResponseWriter, r *http.Request, log *logrus.Entry) ([]byte, error) {
+	subID := r.URL.Query().Get("subid")
+	rg := r.URL.Query().Get("rgn")
+	nicName := r.URL.Query().Get("nic")
+
+	resourceID := strings.TrimPrefix(r.URL.Path, "/admin")
+
+	dbOpenShiftClusters, err := f.dbGroup.OpenShiftClusters()
+
+	if err != nil {
+		//return nil, api.NewCloudError(http.StatusInternalServerError, api.CloudErrorCodeInternalServerError, "", err.Error())
+		log.Fatalf("failed to load cluster from cosmosDB: %v", err)
+	}
```

Comment:
```suggestion
	if err != nil {
		return nil, api.NewCloudError(http.StatusInternalServerError, api.CloudErrorCodeInternalServerError, "", err.Error())
	}
```

I'm assuming this log statement was leftover from local testing.

Also, just pointing out for clarity's sake that an error here doesn't indicate we failed to get the cluster from CosmosDB - it indicates that we failed to get the OpenShiftClusters database client.